Test/UI/LoginPageTest.cs

```diff
@@ -0,0 +1,66 @@
+using Allure.Commons;
+using BusinessObject.Builders;
+using BusinessObject.UI.Pages;
+using Core.Selenium;
+using NUnit.Allure.Attributes;
+
+namespace Test.UI;
+
+public class LoginPageTest : BaseTest
+{
+    private const string ExpectedLogInErrorMessage = "Sorry, there was a problem.";
+    private const string ExpectedLogInErrorDescription = "Email/Login or Password is incorrect. Please try again.";
+    private const string ExpectedValidationErrorMessage = "Email/Login is required.";
+
+    [Test]
+    [Category("UI")]
+    [AllureTag("UI")]
+    [AllureOwner("Igor Kardanov")]
+    [AllureSuite("Login")]
+    [AllureSubSuite("Positive")]
+    [AllureSeverity(SeverityLevel.critical)]
+    public void LoginPositive()
+    {
+        var user = UserBuilder.GetTestRailUser();
+        var loginPage = new LoginPage(Browser.Instance.Driver);
+        var mainPage = loginPage.Open().Login(user);
+        WaitingHelper.WaitUntilUrlToBe(Browser.Instance.Driver, mainPage.Url);
+        Assert.That(Browser.Instance.Driver.Url, Is.EqualTo(mainPage.Url));
```

Comment:
create method in the browser for get url

Browser.Instance.Url;

 where  URL will be => Instance.Driver.Url;

Reply:
Done